server/views/pages/dateAndTime.njk

```diff
@@ -26,10 +27,86 @@
 
     {% if slotsList | length %}
     <p>Showing visit time slots for the next 28 days.</p>
+    <form action="/visit/select-date-and-time/{{ offenderNo }}" method="GET" novalidate>
+      <div class="bapv-visit-date-time-filter">
+        <h3 class="govuk-heading-s">Filter time slots</h3>
+        <div class="govuk-button-group">
+          {{ govukSelect({
```

Comment:
Could get the currently selected values here so they don't always just show defaults?

Reply:
Good point! I'll have a look.